refs/types.proto

```diff
@@ -105,6 +105,18 @@ message Signature {
   bytes key = 1 [json_name = "key"];
   // Signature
   bytes sign = 2 [json_name = "signature"];
+  // Scheme contains digital signature scheme identifier.
+  SignatureScheme scheme = 3 [json_name = "scheme"];
+}
+
+// Signature scheme describes digital signing scheme used for (key, signature) pair.
+enum SignatureScheme {
+  // Unknown.
```

Comment:
Let's add a comment clarifying that `UNKNOWN` will be interpreted according to the defaults specific for a particular structure type.